backend/tracim_backend/lib/mail_notifier/utils.py

```diff
@@ -1,11 +1,12 @@
 class SmtpConfiguration(object):
     """Container class for SMTP configuration used in Tracim."""
 
-    def __init__(self, server: str, port: int, login: str, password: str):
+    def __init__(self, server: str, port: int, login: str, password: str, implicit_ssl: bool):
```

Comment:
its not intuitive that implicit_ssl is a boolean. it should be use_implicit_ssl or something like this

Reply:
i agree.